README.md

```diff
@@ -23,6 +23,7 @@ It is available for install from the [JetBrains Marketplace](https://plugins.jet
  - Navigate into Kubernetes workloads, services, pods and nodes
  - Switching contexts and namespaces within Kubernetes
  - Follow logs from the connected cluster resource
+ - Open a Terminal Shell to a Running Container in the connected cluster
 
```

Comment:
```suggestion
Support [init containers](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) workflow. These are specialized containers that run before app containers in a [Pod](https://kubernetes.io/docs/concepts/workloads/pods/). Init containers can contain utilities or setup scripts not present in an app image
```

Reply:
To me these bullet points should be key points of what's possible. You added 3 full sentences. I'd move the explanations further down to he text even though there's already text about it in follow logs and terminal.

Reply:
I think this work with bullet point.

Reply:
I think that this is far too long for an overview. I especially notice this when reading all the bullet points. It's striking how these are very short but the last one is very verbose.